### DISCUSSION TITLE
寻求yolo_obj_det 和 yolo_edge_det 的pth权重文件

### DISCUSSION THREAD

**Discussion by shuchao97** [RapidAI/TableStructureRec#74] [Q&A]
大佬，你好，看了你很多的关于OCR相关仓库的，对您的编码能力尤其佩服。这些我作品我也实际测试过，感觉效果很好。我目前在做表格的定位，我参考了你的另一库RapidTableDetection,我对yolo_obj_det.onnx和yolo_edge_det.onnx 模型本身测试测试没有问题，测试定位的效果也相当不错。我现在希望能够将onnx模型转化到第三方的推理格式文件，但是你导出onnx版本都太高，我直接通过你提供的onnx模型转化，模型精度损失太高，我希望能够寻求这个两个模型原始的pth文件，祝好

**Comment by Joker1212** [comment]
工程上主要还是学之前Rapid成员的哈，我也是加入不久，有兴趣可以看看RapidLayout的源码，主要是参考它写的，模型文件在 https://www.modelscope.cn/models/jockerK/TableExtractor/files
